### PR TITLE
MinnekDigitalStudio.Cominnek version 2.0.0-alpha.2

### PR DESCRIPTION
**Pull request by isaacismaelx14**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.2 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/82106)

### CODE REVIEW

**Comment by isaacismaelx14** [comment]
@microsoft-github-policy-service agree [company="Minnek Digital Studio"]

**Comment by isaacismaelx14** [comment]
@microsoft-github-policy-service agree company="Minnek Digital Studio"

File: manifests/m/MinnekDigitalStudio/Cominnek/2.0.0-alpha.2/MinnekDigitalStudio.Cominnek.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: MinnekDigitalStudio.Cominnek
+PackageVersion: 2.0.0-alpha.2
+PackageLocale: en-US
+Publisher: Minnek Digital Studio
+PackageName: cominnek
```
Comment (ItzLevvie) [inline manifests/m/MinnekDigitalStudio/Cominnek/2.0.0-alpha.2/MinnekDigitalStudio.Cominnek.locale.en-US.yaml:8]:
```suggestion
PackageName: Cominnek
```

File: manifests/m/MinnekDigitalStudio/Cominnek/2.0.0-alpha.2/MinnekDigitalStudio.Cominnek.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: MinnekDigitalStudio.Cominnek
+PackageVersion: 2.0.0-alpha.2
```
Comment (ItzLevvie) [inline manifests/m/MinnekDigitalStudio/Cominnek/2.0.0-alpha.2/MinnekDigitalStudio.Cominnek.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 2.0.0-alpha.3
```

File: manifests/m/MinnekDigitalStudio/Cominnek/2.0.0-alpha.2/MinnekDigitalStudio.Cominnek.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: MinnekDigitalStudio.Cominnek
+PackageVersion: 2.0.0-alpha.2
```
Comment (ItzLevvie) [inline manifests/m/MinnekDigitalStudio/Cominnek/2.0.0-alpha.2/MinnekDigitalStudio.Cominnek.installer.yaml:5]:
```suggestion
PackageVersion: 2.0.0-alpha.3
```

File: manifests/m/MinnekDigitalStudio/Cominnek/2.0.0-alpha.2/MinnekDigitalStudio.Cominnek.yaml [hunk 4]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.2.0.schema.json
+
+PackageIdentifier: MinnekDigitalStudio.Cominnek
+PackageVersion: 2.0.0-alpha.2
```
Comment (ItzLevvie) [inline manifests/m/MinnekDigitalStudio/Cominnek/2.0.0-alpha.2/MinnekDigitalStudio.Cominnek.yaml:5]:
```suggestion
PackageVersion: 2.0.0-alpha.3
```

File: manifests/m/MinnekDigitalStudio/Cominnek/2.0.0-alpha.2/MinnekDigitalStudio.Cominnek.installer.yaml [hunk 5]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: MinnekDigitalStudio.Cominnek
+PackageVersion: 2.0.0-alpha.2
+Installers:
+- Architecture: x64
+  InstallerType: inno
+  InstallerUrl: https://github.com/Minnek-Digital-Studio/cominnek/releases/download/v2.0.0-alpha.2/cominnek_2.0.0-alpha.2.exe
```
Comment (ItzLevvie) [inline manifests/m/MinnekDigitalStudio/Cominnek/2.0.0-alpha.2/MinnekDigitalStudio.Cominnek.installer.yaml:9]:
```suggestion
  InstallerUrl: https://github.com/Minnek-Digital-Studio/cominnek/releases/download/v2.0.0-alpha.3/cominnek_2.0.0-alpha.3.exe
```

File: manifests/m/MinnekDigitalStudio/Cominnek/2.0.0-alpha.2/MinnekDigitalStudio.Cominnek.installer.yaml [hunk 6]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: MinnekDigitalStudio.Cominnek
+PackageVersion: 2.0.0-alpha.2
+Installers:
+- Architecture: x64
+  InstallerType: inno
+  InstallerUrl: https://github.com/Minnek-Digital-Studio/cominnek/releases/download/v2.0.0-alpha.2/cominnek_2.0.0-alpha.2.exe
+  InstallerSha256: 6695685720437228A3A852713D7CFF0DB998372FE712E3B123C82F135495E0CF
```
Comment (ItzLevvie) [inline manifests/m/MinnekDigitalStudio/Cominnek/2.0.0-alpha.2/MinnekDigitalStudio.Cominnek.installer.yaml:10]:
```suggestion
  InstallerSha256: A554C7EF707192F963223C2C9ACCD49B30851FAAD8065440445CEB0E29D89D04
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Pull request closed without merging**